src/MainWindow/CustomFlow/index.jsx

```diff
@@ -25,20 +25,14 @@ const CustomFlowPage = ({ completedScanId, setCompletedScanId }) => {
     const [isReplay, setIsReplay] = useState(false);
 
     useEffect(() => {
-        console.log("state.scanDetails: ", state.scanDetails);
-        if (state.scanDetails) {
-          console.log("entered scan details");
+        if (state && state.scanDetails) {
```

Comment:
```suggestion
        if (state?.scanDetails) {
```

Nitpick: but we can refactor this next time